plugins/modules/ec2_scaling_policy.py

```diff
@@ -355,28 +531,54 @@ def main():
     step_adjustment_spec = dict(
         lower_bound=dict(type='int'),
         upper_bound=dict(type='int'),
-        scaling_adjustment=dict(type='int', required=True))
+        scaling_adjustment=dict(type='int', required=True)
+    )
+
+    # Spec for TargetTrackingScaling policies
+    target_tracking_spec = dict(
```

Comment:
You're missing some types in this spec.  I find it cleaner to break out the options into separate vars as this example, it's a stylistic preference however you prefer it. 
```
    # Specs for TargetTrackingScaling policies
    predefined_metric_spec = dict(
        predefined_metric_type=dict(type='str', choices=['ASGAverageCPUUtilization',
                                                         'ASGAverageNetworkIn',
                                                         'ASGAverageNetworkOut',
                                                         'ALBRequestCountPerTarget']),
        resource_label=dict(type='str')
    )
    customized_metric_spec = dict(
        metric_name=dict(type='str', required=True),
        namespace=dict(type='str', required=True),
        statistic=dict(type='str', required=True, choices=['Average', 'Minimum', 'Maximum', 'SampleCount', 'Sum']),
        dimensions=dict(type='list', elements='str'),
        unit=dict(type='str', required=True)
    )
```
Then below in the arg_spec:
```
        target_tracking_spec=dict(options=dict(
            disable_scalein=dict(type='bool'),
            target_value=dict(type='float', required=True),
            predefined_metric_spec=dict(type='dict',
                                        options=predefined_metric_spec),
            customized_metric_spec=dict(type='dict',
                                        options=customized_metric_spec)
        )),
```